polyglot/piranha/src/lib.rs

```diff
@@ -89,7 +90,7 @@ pub fn execute_piranha(
 
   let source_code_units = flag_cleaner.get_updated_files();
 
-  if should_rewrite_files {
+  if *configuration.should_rewrite_files() {
```

Comment:
i guess running `cargo clippy` will fix this

Reply:
I didn't see anything returned by `cargo clippy` for this line, but what did you have in mind?

Reply:
aah ok. I was thinking that we don't really need to dereference `configurations`. But its okay. Maybe I am confused.